src/main/java/com/zufar/onlinestore/payment/api/PaymentApi.java

```diff
@@ -0,0 +1,21 @@
+package com.zufar.onlinestore.payment.api;
+
+import com.stripe.exception.SignatureVerificationException;
+import com.stripe.exception.StripeException;
+import com.zufar.onlinestore.payment.dto.CreatePaymentDto;
+import com.zufar.onlinestore.payment.dto.CreatePaymentMethodDto;
+import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
+import com.zufar.onlinestore.payment.dto.PaymentDetailsWithTokenDto;
+import com.zufar.onlinestore.payment.exception.PaymentNotFoundException;
+
+public interface PaymentApi {
+
+    PaymentDetailsWithTokenDto createPayment(CreatePaymentDto createPaymentDto) throws StripeException;
```

Comment:
Add java doc to all methods where you describe method description, method parameters, method throws exceptions

Reply:
Done